olcPGEX_AnimatedSprite.h

```diff
@@ -69,16 +69,17 @@ namespace olc
 		void SetState(std::string newState);
 		std::string GetState();
 		void Draw(float fElapsedTime, olc::vf2d position);
-		void AddState(std::string stateName, std::vector<std::string> imagePaths);
-		void AddState(std::string stateName, std::vector<olc::vi2d> spriteLocations);
+		void AddState(std::string stateName, float frameDuration, std::vector<std::string> imagePaths);
```

Comment:
I think it would make sense to have these as additional overloads to the `AddState` method - that way the default can still be used without having to define it for every single state if it isn't needed.

Reply:
I've added that in the last commits.